src/main/java/jenkins/plugins/git/GitSCMSource.java

```diff
@@ -404,6 +409,16 @@ public List<SCMSourceTrait> getTraits() {
         return traits;
     }
 
+    /**
+     *
+     * @param credentialsId any credentials (can be {@code null}
+     * @param remoteUrl the git remote url
+     * @return will return {@code false} if using any credentials with a non TLS protocol with FIPS mode activated @see {@link FIPS140#useCompliantAlgorithms()}
+     */
+    public static boolean isFIPSLts(String credentialsId, String remoteUrl) {
```

Comment:
it took me a bit to really read the return. Maybe a more concrete name?
```suggestion
    public static boolean isFIPSLtsCompliant(String credentialsId, String remoteUrl) {
```

Reply:
good idea. fixed in rev f9668af2

Reply:
typo fixed with f6470311